### PR TITLE
[BakaBT] Added categories for tracker

### LINKED ISSUE #5664: [BakaBT] Every entry has season 1 appended to it. Also audio and manga are being shown under category tv/anime.
When searching for anime in BakaBT the word season 1 always get appended to the title. Also manga and audio are being shown under the category of tv/anime. 
When searching for anime in Sonarr these torrents almost never show up or gets selected for download.
Please see the attached screenshot to see what happens.
I am currently on the latest version v0.11.487.
Example

### PR DESCRIPTION
**Pull request by jellofishi**
Attempt to partially resolve #5664 regarding categories not being correctly set on results.

### CODE REVIEW

File: src/Jackett.Common/Indexers/BakaBT.cs [hunk 1]
```diff
@@ -46,6 +48,21 @@ public BakaBT(IIndexerConfigurationService configService, Utils.Clients.WebClien
             Encoding = Encoding.UTF8;
             Language = "en-us";
             Type = "private";
+
+            InitCategoryMappings();
+        }
+
+        private void InitCategoryMappings()
+        {
+            AddCategoryMapping(1, TorznabCatType.TVAnime, "Anime Series");
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/BakaBT.cs:57]: Move this code at the end of `ConfigurationDataBasicLogin` and remove the `InitCategoryMappings` method.

File: src/Jackett.Common/Indexers/BakaBT.cs [hunk 2]
```diff
@@ -199,6 +219,39 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
...
+        private ICollection<int> GetNextCategory(IElement row, ICollection<int> currentCategories)
+        {
+            string nextCategoryName = GetCategoryName(row);
+            if (nextCategoryName != null)
+            {
+                currentCategories = MapTrackerCatDescToNewznab(nextCategoryName);
+                if (currentCategories.Count == 0)
+                {
+                    return defaultCategories;
+                }
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/BakaBT.cs:231]: remove braces

File: src/Jackett.Common/Indexers/BakaBT.cs (hunk 1 above)
Comment (jellofishi) [reply src/Jackett.Common/Indexers/BakaBT.cs:57]: will get around to it on weekend

File: src/Jackett.Common/Indexers/BakaBT.cs (hunk 2 above)
Comment (jellofishi) [reply src/Jackett.Common/Indexers/BakaBT.cs:231]: will get around to it on weekend

**Comment by jellofishi** [comment]
> Did you tested it with several searches?

have tested with a few searches, one of each category and some with multiple categories

File: src/Jackett.Common/Indexers/BakaBT.cs (hunk 1 above)
Comment (jellofishi) [reply src/Jackett.Common/Indexers/BakaBT.cs:51]: Done

File: src/Jackett.Common/Indexers/BakaBT.cs (hunk 2 above)
Comment (jellofishi) [reply src/Jackett.Common/Indexers/BakaBT.cs]: Done

**Comment by garfield69** [comment]
Jackett 0.16.976

**Comment by NGRichter** [comment]
Hi, I don't know if this is the right place to ask this but someone changed the categories.
Right now it is:

            AddCategoryMapping(1, TorznabCatType.TVAnime, "Anime/Series")  
            AddCategoryMapping(2, TorznabCatType.TVAnime, "Anime/OVA")  
            AddCategoryMapping(3, TorznabCatType.AudioOther, "Audio/Soundtrack")
            AddCategoryMapping(4, TorznabCatType.BooksComics, "Books/Manga");
            AddCategoryMapping(5, TorznabCatType.TVAnime, "Anime/Movie");
            AddCategoryMapping(6, TorznabCatType.TVOther, "TV/Live Action");
            AddCategoryMapping(7, TorznabCatType.BooksOther, "Books/Artbook");
            AddCategoryMapping(8, TorznabCatType.AudioVideo, "Audio/Music Video");
            AddCategoryMapping(9, TorznabCatType.BooksEBook, "Books/Light Novel");

While I think it should be different. I think @jellofishi correctly did the category names in commit 0bc64c6 mentioned above.

            AddCategoryMapping(1, TorznabCatType.TVAnime, "Anime Series");
            AddCategoryMapping(2, TorznabCatType.TVAnime, "OVA");
            AddCategoryMapping(3, TorznabCatType.AudioOther, "Soundtrack");
            AddCategoryMapping(4, TorznabCatType.BooksComics, "Manga");
            AddCategoryMapping(5, TorznabCatType.TVAnime, "Anime Movie");
            AddCategoryMapping(6, TorznabCatType.TVOTHER, "Live Action");
            AddCategoryMapping(7, TorznabCatType.BooksOther, "Artbook");
            AddCategoryMapping(8, TorznabCatType.AudioVideo, "Music Video");
            AddCategoryMapping(9, TorznabCatType.BooksEbook, "Light Novel");

**Pull request merged**